### PR TITLE
[FLINK-10803] Update the documentation to include changes to the S3 connector.

### PR DESCRIPTION
**Pull request by kl0u**
This PR simply documents already introduced changes.

To build the docs and verify the change go to the `FLINK_DIR/docs/` directory and run `./build_docs.sh -p`. The documentation will be available at `localhost:4000`.

### CODE REVIEW

**Comment by kl0u** [comment]
R @GJL , @twalthr , @aljoscha

File: docs/dev/connectors/streamfile_sink.md [hunk 1]
```diff
@@ -24,16 +24,24 @@ under the License.
 -->
 
 This connector provides a Sink that writes partitioned files to filesystems
-supported by the Flink `FileSystem` abstraction. Since in streaming the input
-is potentially infinite, the streaming file sink writes data into buckets. The
-bucketing behaviour is configurable but a useful default is time-based
+supported by the [Flink `FileSystem` abstraction]({{ site.baseurl}}/ops/filesystems.html).
+
+<span class="label label-danger">Important Note</span>: For S3, the `StreamingFileSink` 
+supports only Hadoop (not Presto). In case your job uses the `StreamingFileSink` to write to S3 but
```
Comment (zentol) [inline docs/dev/connectors/streamfile_sink.md]: does this page contain a link to presto somewhere? If not, please add one.

**Comment by aljoscha** [comment]
I think the changes look good modulo the comments @zentol  and I had.

**Comment by kl0u** [comment]
Thanks a lot for the comments @zentol and @aljoscha ! I will integrate them and merge.

**Pull request closed without merging**